scripts/deploy_assisted_service.sh

```diff
@@ -117,9 +117,15 @@ else
     print_log "Updating assisted_service params"
 
     if [ "${DEBUG_SERVICE}" == "true" ]; then
-        print_log " Patching assisted service image with a debuggable code "
-        (cd assisted-service/ && skipper --env-file ../skipper.env make update-local-image -e LOCAL_ASSISTED_ORG=$(minikube ip):5000 -e CONTAINER_BUILD_EXTRA_PARAMS="--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file")
+        # Forward the minikube registry addon k8s service to the host to push the debug image using localhost:5000
+        kubectl port-forward service/registry -n kube-system 5000:80 > /dev/null 2>&1 &
+        # Set the local registry to the minikube registry (used by the assisted-service update-local-image target)
+        export LOCAL_ASSISTED_ORG=localhost:5000
+        print_log "Patching assisted service image with a debuggable code "
+        (cd assisted-service/ && skipper --env-file ../skipper.env make update-local-image -e CONTAINER_BUILD_EXTRA_PARAMS="--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file")
```

Comment:
export `CONTAINER_BUILD_EXTRA_PARAMS` like you did for `LOCAL_ASSISTED_ORG`.

Reply:
It used only once

Reply:
I just think it would look nicer.. but not critical.